gradle.properties

```diff
@@ -33,4 +33,5 @@ pomLicenceName=The MIT License (MIT)
 pomLicenceUrl=http://opensource.org/licenses/MIT
 pomLicenceDist=repo
 pomDeveloperId=thyrlian
-pomDeveloperName=Jing Li
\ No newline at end of file
+pomDeveloperName=Jing Li
+android.useAndroidX=true
```

Comment:
It's better to have segregation with the above POM attributes.  But this is a minor thing.